src/main/java/appeng/parts/networking/PartQuartzFiber.java

```diff
@@ -19,7 +19,10 @@
 package appeng.parts.networking;
 
 
+import java.util.Collection;
+import java.util.Collections;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Collections'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AUselessImportCheck)